games-simulation/corsix_th/corsix_th-0.62.recipe

```diff
@@ -0,0 +1,90 @@
+SUMMARY="Open source clone of Theme Hospital"
+DESCRIPTION="
+CorsixTH aims to reimplement the game engine of Theme Hospital, and be able to \
+load the original game data files. This means that you will need a purchased \
+copy of Theme Hospital, or a copy of the demo, in order to use CorsixTH. After \
+most of the original engine has been reimplemented in open source code, the \
+project will serve as a base from which extensions and improvements to the \
+original game can be made.
+"
+HOMEPAGE="https://github.com/CorsixTH"
+COPYRIGHT="2005-2018 Corsix"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/CorsixTH/CorsixTH/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="b219270310255493c881a09bb4a5569f34a72cdaf3c3be920c1036a2450317ce"
+SOURCE_FILENAME="CorsixTH-$portVersion.tar.gz"
+SOURCE_DIR="CorsixTH-$portVersion"
+PATCHES="corsix_th-$portVersion.patchset"
+ADDITIONAL_FILES="corsix_th.rdef.in"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+SECONDARY_ARCHITECTURES="x86"
+fi
+
+PROVIDES="
+	corsix_th$secondaryArchSuffix = $portVersion
+	cmd:corsix_th$secondaryArchSuffix = $portVersion
```

Comment:
When using $prefix the binary will go into $prefix/bin and shouldn't get the $secondaryArchSuffux
```diff
-	cmd:corsix_th$secondaryArchSuffix = $portVersion
+	cmd:corsix_th = $portVersion
```